deploy/osd-rebalance-infra-nodes/07-osd-rebalance-infra-nodes.ConfigMap.yaml

```diff
@@ -178,4 +178,5 @@ data:
     waitRunningPods splunk-heavy-forwarder openshift-security name
 
     echo "INFO: Rebalancing misscheduled pods from openshift- namespaces"
-    infraPodsMisscheduled  openshift-cloud-ingress-operator openshift-custom-domains-operator openshift-image-registry openshift-managed-node-metadata-operator openshift-monitoring openshift-must-gather-operator openshift-network-operator openshift-ocm-agent-operator openshift-osd-metrics openshift-rbac-permissions openshift-route-monitor-operator openshift-splunk-forwarder-operator openshift-velero
+    # disable openshift-image-registry before https://issues.redhat.com/browse/OSD-15340 is fixed
```

Comment:
From my understanding, this effectively disables the rebalacing of `openshift-image-registry` as a whole. What are the implications on a normal cluster here? 
Would it make sense to only disable the rebalancing for image registry if there's more than 2 infra nodes?